src/sass/chrome.scss

```diff
@@ -47,145 +41,66 @@ aside {
 
 // Page temp styles
 .pf-c-page {
-    --pf-c-page__sidebar--Transition: all 500ms cubic-bezier(.42, 0, .58, 1);
-    --pf-m-emphasis--FontSize: var(--pf-global--FontSize--xl);
-
-    .ins-c-chrome__global-filter {
-        padding: var(--pf-global--spacer--md) var(--pf-global--spacer--lg);
-        background: var(--pf-global--BackgroundColor--100);
```

Comment:
@epwinchell the white background is missing from the global filter:

![Screenshot from 2022-02-14 11-49-53](https://user-images.githubusercontent.com/22619452/153850203-3595a83b-bdb9-4919-9be1-21897846e4c7.png)

The issue is present in stage beta.